llm_on_ray/finetune/finetune.py

```diff
@@ -68,23 +71,23 @@ def get_accelerate_environment_variable(mode: str, config: Union[Dict[str, Any],
             "FSDP_SYNC_MODULE_STATES": "true",
             "ACCELERATE_MIXED_PRECISION": mixed_precision,
         },
-        "GPU_DEEPSPEED": {
+        "gpu_DEEPSPEED": {
             "ACCELERATE_USE_CPU": "false",
             "ACCELERATE_USE_XPU": "true",
             "ACCELERATE_USE_IPEX": "true",
             "ACCELERATE_USE_DEEPSPEED": "true",
             "ACCELERATE_MIXED_PRECISION": mixed_precision,
         },
-        "HPU_DDP": {
+        "hpu_DDP": {
             "ACCELERATE_USE_CPU": "false",
             "ACCELERATE_USE_XPU": "false",
             "ACCELERATE_USE_IPEX": "false",
             "ACCELERATE_MIXED_PRECISION": mixed_precision,
         },
     }
-    if mode not in mode_env_vars:
+    if device_accelerate_mode not in mode_env_vars:
         raise ValueError(f"accelerate mode must be one of {list(mode_env_vars.keys())}")
-    return mode_env_vars[mode]
+    return mode_env_vars[device_accelerate_mode]
```

Comment:
The error message above is no longer correct because accelerate mode can only be CPU, GPU or Deepspeed now. Can you please print something like Accelerate mode xxx is not supported on device xxx?

Reply:
device is ["cpu", "gpu", "hpu"]
accelerate mode is ["DDP", "FSDP", "DEEPSPEED"]

Reply:
cpu only supports DDP, right? so FSDP/Deepspeed is not supported on CPU.

Reply:
yes, if device set to CPU, accelerate mode set to FSDP, main function will raise error message